src/tr.i18n.js

```diff
@@ -257,16 +255,15 @@ module.exports = {
   },
   'menu.variation': {
     'Add Variation': 'Varyasyon Ekle',
-    'The root node cannot have sibling nodes.':
-      'Kök düğümün kardeş düğümleri olamaz'
+    'The root node cannot have sibling nodes.': 'Kök düğümde kardeş düğümler olamaz.'
   },
   'menu.engineAction': {
     'Start': 'Başlat',
     'Stop': 'Durdur',
-    'Detach': 'Çıkart',
-    'Synchronize': 'Senkronize',
+    'Detach': 'Ayır',
```

Comment:
After spending more time by using program i have realized that "Bağlantıyı Kes" is more proper translation.